### ISSUE TITLE
Landing on a player teleports them inside

### ISSUE THREAD

**Issue by ask-compu** [DarkholmeTenk/TardisModIssues#549]
why not do this instead of the landing on a player damages them thing?

**Comment by FoxPotato** [comment]
From what Dark told me it's not possible. I don't recall why, but it has something to do with being damaged and getting pushed away.

**Comment by HalJordan** [comment]
/It would make more sense with cannon, but I'm not sure you can effect the other players in that way. I'd also rather they put more effort into getting it ready for 1.9/1.10 than adding something like that.

**Comment by FoxPotato** [comment]
@HalJordan #458 

also, you can land on mobs to teleport them in

**Comment by AlienXtream** [comment]
yeah, that happened to my friend on a lan game. he landed on a villager and i said "and now it's inside the tardis". they didn't catch on till they entered their tardis next XD

**Comment by handicraftsman** [comment]
Not issue, works as decided (reason - same idea was implemented in Dr. Who tv series. Just watch it)

**Comment by AlienXtream** [comment]
@handicraftsman um.... what??? that made zero sense
